core/tools/blueos_startup_update/blueos_startup_update.py

```diff
@@ -448,6 +450,10 @@ def run_command_is_working():
 
 def main() -> int:
     start = time.time()
+    # check if boot_loop_detector exists
+    if os.path.isfile(BOOT_LOOP_DETECTOR):
+        logger.warning("It seems we are in a boot loop, skipping patches.")
```

Comment:
"It seems that we are starting from a previous startup configuration, skipping patches."

Reply:
that sounds... vague
what about "It seems we the startup patches were just applied on the last boot. skipping patches..." ?

Reply:
Yep